### ISSUE TITLE
Sonar seems to have different configuration for merging to main vs merging to develop

### ISSUE THREAD

**Issue by AleJo2995** [oscal-compass/compliance-trestle#1569]
## Describe the bug

Quite recently we have made a release to the latest version of the code and before merging changes into develop, Sonar quality checks were passing successfully. When creating the PR that goes from develop to main, Sonar checks started to throw an unexpected error that was not showing-off before when merging to develop.

Now, it seems that there might some mismatch of configuration between the two places, so this needs further investigation

## Expected behavior

Both configurations should match to avoid getting to merge to main with problems we didn´t encounter previously.

## Environment

- OS: \[e.g. iOS\]
- Python version:
- Installed packages:

**Comment by jpower432** [comment]
As discussed with the team, all pull request into develop should be scanned with sonar. There can only be one MAIN branch and currently that is set to the `develop` branch. The `main` branch can be tracked as a long lived branch.

**Comment by butler54** [comment]
It looks like Sonarqube is not configured to run against remote repos: https://github.com/oscal-compass/compliance-trestle/blob/8744cee2beb966e99b338e71ccf723b805b9b4d7/.github/workflows/python-test.yml#L160

Doing a bit of a google the answer is because it's janky to make it work e.g.: https://community.sonarsource.com/t/sonar-cannot-be-run-on-pr-from-a-fork/69229. 

Looking at the [CNCF hosted services
](https://contribute.cncf.io/resources/project-services/hosted-tools/) FOSSA and LFX I think it might make sense to migrate rather than mess with Sonarqube.

**Comment by jpower432** [comment]
@butler54 I agree and this topic was dicussed at the Oversight Committee meeting today. During CNCF sandbox on-boarding, we chose Synk for license compliance scanning and I took an action item to understand if [LFX Security](https://security.lfx.linuxfoundation.org/#/) (powered in part by Synk) could fill the requirement we are using Sonarqube for right now. I think this would also reduce friction in the PR workflow for forks.
